bindings/rust/s2n-tls/src/testing/s2n_tls.rs

```diff
@@ -892,4 +892,55 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn key_updates() -> Result<(), Error> {
+        let config = {
+            let config = config_builder(&security::DEFAULT_TLS13)?;
+            config.build()?
+        };
+
+        let server = {
+            let mut server = crate::connection::Connection::new_server();
+            server.set_config(config.clone())?;
+            Harness::new(server)
+        };
+
+        let client = {
+            let mut client = crate::connection::Connection::new_client();
+            client.set_config(config)?;
+            Harness::new(client)
+        };
+
+        let pair = Pair::new(server, client);
+        let mut pair = poll_tls_pair(pair);
```

Comment:
This is pretty verbose for just doing a basic handshake with no customization. Can't you just do:
```
let config = config_builder(&security::DEFAULT_TLS13)?.build()?;
let pair = poll_tls_pair(tls_pair(config));
let server = pair.server.0.connection;
let client = pair.client.0.connection;
```

Reply:
Eh "pair.server.0.connection.as_ref()" isn't pretty, but it's not the end of the world. The test is much cleaner now!